pkg/cache/remote.go

```diff
@@ -38,7 +39,12 @@ func NewRemoteCache(url string, customHeaders http.Header, insecure bool) cache.
 
 // PutArtifact sends artifact to remote client
 func (c RemoteCache) PutArtifact(imageID string, artifactInfo types.ArtifactInfo) error {
-	_, err := c.client.PutArtifact(c.ctx, rpc.ConvertToRPCArtifactInfo(imageID, artifactInfo))
+	var _ *emptypb.Empty
```

Comment:
What is this for?

Reply:
The response that is not in used. What should I do instead?

Reply:
What's going to happen if we just remove this line?

Reply:
I need to add it to the inner close function

Reply:
Did you see any errors?

Reply:
updated